mac

```diff
@@ -216,7 +216,7 @@ append_web_dependencies() {
     brew "elixir"
 
     # Devops
-    cask "docker"
+    cask "docker" unless File.directory?("/Applications/Docker.app")
```

Comment:
not sure if it's needed, just to save some Internet bandwidth in case docker desktop is already installed!

Reply:
This would be required indeed as Hoembreak install application into its own directory. But again it means it would mean to have this check on all apps 😅  Maybe reworking the function `cask` would be better so that it does this check each for each app:

```
# the second parameter would be the Mac app name
cask "docker", "Docker" 
cask "docker", "JetBrains Toolbox" 
```

In any case, this is outside the scope of this PR which is about "Update mas-cli package source" so it should be part of another PR.